docs/installation/mac.md

```diff
@@ -274,85 +326,87 @@ The next exercise demonstrates how to do this.
+| init           | create           | Creates a new docker host.                               |
+| up             | start            | Starts a stopped machine.                                |
+| ssh            | ssh              | Runs a command or interactive ssh session on the machine.|
+| save           | -                | Not applicable.                                          |
+| down           | stop             | Stops a running machine.                                 |
+| poweroff       | stop             | Stops a running machine.                                 |
+| reset          | restart          | Restarts a running machine.                              |
+| config         | inspect          | Prints machine configuration details.                    |
+| status         | ls               | Lists all machines and their status.                     |
+| info           | inspect          | Displays a machine's details.                            |
+| ip             | ip               | Displays the machine's ip address.                       |
+| shellinit      | env              | Displays shell commands needed to configure your shell to interact with a machine |
+| delete         | rm               | Removes a machine.                                       |
+| download       | -                | Not applicable.                                          |
+| upgrade        | upgrade          | Upgrades a machine's Docker client to the latest stable release. |
 
 
-## Uninstallation 
+## Upgrade Docker Toolbox
 
-1. Go to the [boot2docker/osx-installer ](
-   https://github.com/boot2docker/osx-installer/releases/latest) release page. 
+To upgrade Docker Toolbox, download an re-run [the Docker Toolbox
+installer](https://github.com/docker/toolbox/releases/tag/v1.7.0).
 
-2. Download the source code by clicking `Source code (zip)` or
-   `Source code (tar.gz)` in the "Downloads" section.
 
-3. Extract the source code.
+## Uninstall Docker Toolbox 
 
-4. Open a terminal on your local machine.
+To uninstall, do the following:
 
-5. Change to the directory where you extracted the source code:
+1. List your machines.
 
-        $ cd <path to extracted source code>
+      $ docker-machine ls
+      NAME                ACTIVE   DRIVER       STATE     URL                         SWARM
+      dev                 *        virtualbox   Running   tcp://192.168.99.100:2376   
+      my-docker-machine            virtualbox   Stopped                               
+      my-docker-vm                 virtualbox   Stopped  
 
-6. Make sure the uninstall.sh script is executable:
+2. Remove each machine.
 
-        $ chmod +x uninstall.sh
+      $ docker-machine rm dev
+      Successfully removed dev
+    
+   Removing a machine deletes its VM from virtual box and from the
+   `~/.docker/machine/machines` directory.
 
-7. Run the uninstall.sh script:
+3. Remove the Docker CLI and Kitematic from your "Applications" folder.
```

Comment:
Think this needs to be indented more, because GitHub renders this as regulars text (not code)